source/test/F0.Minesweeper.Components.Tests/Pages/Game/Modules/CellTests.razor

```diff
@@ -1,18 +1,20 @@
 @inherits TestContext
 
 @code {
-	private readonly Mock<ICellStatusManager> cellStatusManagerMock;
+    private readonly Mock<ICellStatusManager> cellStatusManagerMock;
```

Comment:
_Spaces_ instead of _tabs_ in parts of this file.
VS2022s Razor Editor still does not respect the `.editorconfig` file 😢.

Reply:
I should've gotten all of those nasty spaces now :)